src/NServiceBus.Core.Tests/Scheduler/ScheduleTests.cs

```diff
@@ -83,7 +83,7 @@ public Task Subscribe(Type eventType, SubscribeOptions options)
                 throw new NotImplementedException();
             }
 
-            public Task Unsubscribe(Type eventType, UnsubscribeOptions options)
+            public Task Unsubscribe(Type eventType, UnsubscribeOptions unsubscribeOptions)
```

Comment:
Most of the others use "options" so we should either change all the methods or rename this one back to "options"?

Reply:
We saw another method on the `IMessageSession` use `publishOptions` thus we applied this but it seems most indeed use `options`. Updated the whole interface to use the full name